charts/substra-backend/templates/_helpers.tpl

```diff
@@ -90,6 +90,8 @@ Usage:
 {{ include "common.tplvalues.render" ( dict "value" .Values.path.to.the.Value "context" $) }}
 */}}
 {{- define "common.tplvalues.render" -}}
+    {{ printf "Value: %v" .value | indent 2 }}
+    {{ printf "Context: %v" .context | indent 2 }}
```

Comment:
Is this still needed? I don't understand what it does 😅

Reply:
Sorry, I forgot those I was used for debugging 😄 I'll remove it